phpunit.xml.dist

```diff
@@ -36,6 +36,21 @@
 		<file>tests/Common/BranchWorkspacesTest.php</file>
 	</testsuite>
 
+	<testsuite name="temp-devbranch-only">
+		<file>tests/Common/BranchAwareClientTest.php</file>
+		<file>tests/Common/BranchBucketsTest.php</file>
+		<file>tests/Common/BranchComponentTest.php</file>
+		<file>tests/Common/BranchEventsTest.php</file>
+		<file>tests/Common/ClientProviderTest.php</file>
+		<file>tests/Common/ComponentsEventsTest.php</file>
+		<file>tests/Common/ComponentsTest.php</file>
```

Comment:
zvážil bych jestli tests/Common/EventsTest.php tam nedat taky když budem měnit nějaké ty eventy s branchid

Reply:
Ale případně se to dá přidat v rámci toho tasku až...

Reply:
Pridam ho tam hned.